backend/clubs/models.py

```diff
@@ -698,6 +698,31 @@ def send_approval_email(self, request=None, change=False):
                 reply_to=settings.OSA_EMAILS + [settings.BRANDING_SITE_EMAIL],
             )
 
+    def get_latest_and_latest_approved_versions(self):
+        """
+        Returns the latest and latest approved versions of club key fields
+        [name, description, image] contents that trigger reapproval.
+        """
+        latest_approved_version = (
+            self.history.filter(approved=True).order_by("-history_date").first()
```

Comment:
[Looks like the `history_date` field is indexed](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://django-simple-history.readthedocs.io/en/latest/historical_model.html%23:~:text%3DBy%2520default%2520the%2520history_date%2520field,this%2520behavior%2520using%2520settings.py.&ved=2ahUKEwjdz7njqeuGAxVMMVkFHajuCxEQFnoECBIQAw&usg=AOvVaw3Kwcm4ucYvSbX7gCG3V-OX), in which case it'd be cheaper to order first and filter second. I'm not sure if the ORM does this under the hood, but it's worth looking into the queries being generated. [This could be a good starting point](https://b0uh.github.io/django-show-me-the-sql.html).

Reply:
Oh, I thought filtering first reduces the amount of data to order, making it cheaper?

Reply:
Looks like Django filters first